bitcoin-magazine/bitcoin-explained/bitcoin-core-24.0.md

```diff
@@ -0,0 +1,1150 @@
+So what miners do is ultimately up to miners.
+You have no control over that.
+And Bitcoin Core can change things in the software, but miners will do whatever they do, because what the mempool does is not consensus.
+
+However, the nodes will relay transactions.
+And so you can change the nodes to say, well, I'm gonna broadcast some transactions and not gonna rebroadcast other transactions.
+And so this new rule, opt-in replaced by fee, opt-in RBF, basically said that normally we only broadcast the first version of the transaction we see, regardless of the second one.
+Doesn't matter if the second one pays more fees.
+But if you put a flag in a transaction that says, I want to opt-in to replace by fee, then nodes will refer, will relay transactions that pay a higher fee only and a bunch of other constraints.
+And so this gives the recipient some assurance, not much, but some assurance that if this flag is present, this transaction could be replaced anytime.
+You really have to wait for it to confirm.
+If it doesn't have this flag, it might still disappear.
+It's just a little bit less likely.
+
+Aaron van Wirdum: 00:17:35
+
+Yeah, I want to clarify one thing because you mentioned double spending in this context.
+And you are of course right that this can be used to double spend until it's included in a block, as you said.
+But the main purpose or the main idea for using a flag, for example, is to increase the fee on your own transaction, right?
+
+Sjors Provoost: 00:17:54
+
+I don't know how long that idea has been around because the idea of being stuck in a mempool, that concept didn't even exist until 2015 or 2016.
+That was never a problem.
+
+## RBF summary
+Aaron van Wirdum: 00:18:04
+
+But it was the flag was introduced around that time and definitely in the context of that, that was the debate.
+Yeah, I'm sure about that.
+So to summarize that, At least the idea for the flag was you send a transaction, but the mempool was full, and therefore your transaction is not confirming.
+Now with the RBF flag, you can basically resend the same transaction with a higher fee, and therefore, because the flag is included, nodes will actually forward it to miners and it can be included in a block.
+
+Now you don't actually need to include the flag anymore.
+If this setting is turned on?
+
+Sjors Provoost: 00:18:44
+
+Yeah, I think that if you turn this setting on, then the flag does no longer have to be in there, it'll relate anyway so then I'm guessing there are other rules though because I assume the fee has to go up but I'm not sure about that
+
+Aaron van Wirdum: 00:18:56
+
+So in my case if so far I was running a Bitcoin Core node let's say what was the previous one 23 of course So far I was running Bitcoin Core 23 and then my node would not forward a transaction even if it had a higher fee and it conflicted with the previous transaction.
```

Comment:
Here you can see a self-corrections where the speaker revises their own words for accuracy or to add detail. In this case you should include only the most clear and final version of what they say.
```suggestion
So in my case, if so far I was running Bitcoin Core 23, my node would not forward a transaction even if it had a higher fee and it conflicted with the previous transaction.
```

Reply:
This is one of the fixes that really improves readability